File: nohole/Models/BlurStyle.swift
import Foundation

enum BlurStyle: String, CaseIterable, Identifiable {
    case gaussian = "Gaussian Blur"
    case pixelate = "Pixelate"
    case solidBlack = "Solid Black"
    
    var id: String { rawValue }
    
    var iconName: String {
        switch self {
        case .gaussian: return "aqi.medium"
        case .pixelate: return "squareshape.split.3x3"
        case .solidBlack: return "rectangle.fill"
        }
    }
}

File: android/app/src/main/kotlin/com/noglasshole/app/data/BlurStyle.kt
package com.noglasshole.app.data

enum class BlurStyle(val key: String, val displayName: String) {
    GAUSSIAN("gaussian", "Gaussian Blur"),
    PIXELATE("pixelate", "Pixelate"),
    SOLID_BLACK("solid_black", "Solid Black");

    companion object {
        fun fromKey(key: String): BlurStyle =
            entries.firstOrNull { it.key == key } ?: GAUSSIAN
    }
}
